__tests__/index.ts

```diff
@@ -338,7 +338,7 @@ describe('happy path', () => {
   createTest({
     testName: 'codemodArgs parseArgs is passed to postProcess',
     fixtureName: 'parse-args',
-    spawnArgs: ['--codemod', path.join('codemod', 'index.ts'), '*.js', '--', '--requiredFlag'],
+    spawnArgs: ['--codemod', path.join('codemod', 'index.ts'), '*.js', '--jsonOutput', '--', '--requiredFlag'],
```

Comment:
Bug: this test parsed the logs with `getJsonLogs`, but didn't actually pass the `--jsonOutput` option.